sci-visualization/mantid/files/mantid-3.3.0-FindOpenCascade.patch

```diff
@@ -0,0 +1,84 @@
+diff --git a/Build/CMake/FindOpenCascade.cmake b/Build/CMake/FindOpenCascade.cmake
+index aefa011..346678d 100644
+--- a/Build/CMake/FindOpenCascade.cmake
++++ b/Build/CMake/FindOpenCascade.cmake
+@@ -4,13 +4,6 @@
+ # OPENCASCADE_FOUND If false, do not try to use OPENCASCADE
+ 
+ find_path ( OPENCASCADE_INCLUDE_DIR Standard.hxx PATHS
+-                /usr/include/opencascade
+-                /usr/include/oce
+-                /usr/local/include/opencascade
+-                /usr/local/include/oce
+-                /usr/local/inc
+-                /opt/OpenCASCADE/inc
+-                ${CMAKE_INCLUDE_PATH}/OpenCascade
+                 $ENV{CASROOT}/inc
+ )
+ 
+@@ -18,50 +11,56 @@ if ( WIN32 )
+   add_definitions ( -DWNT )
+ endif ( WIN32 )
+ 
+-set ( OC_REDHAT_RPM /opt/OpenCASCADE/lib64 )
++find_path ( OPENCASCADE_LIBRARY_DIR libTKernel.so PATHS
++               $ENV{CASROOT}/lib64
```

Comment:
OK, found it. There still needs to be a patch to replace the `set (OC_REDHAT_RPM ...` with a `find_path` or a `set( ... CACHE)` or something else that can be overridden with a `-D...` command-line argument.